tools/sigclient/pkg/ingest/ingest.go

```diff
@@ -224,7 +223,7 @@ func runIngestion(iType IngestType, rdr utils.Generator, wg *sync.WaitGroup, url
 	t.MaxIdleConns = 500
 	t.MaxConnsPerHost = 100
 	t.MaxIdleConnsPerHost = 100
-	preGeneratedSeriesLength := uint64(len(preGeneratedSeries))
+	seriesId := uint64(0)
```

Comment:
Are you trying to set the global `seriesId` here? Using `:=` will make a new local variable called `seriesId`

Reply:
I removed the local var, it was unnecessary since we only need the global variable and the default value is 0.